data/bash-completion/fwupdtool

```diff
@@ -64,22 +64,22 @@ _fwupdtool_opts=(
 _show_filters()
 {
 	local flags
-	flags="$(command @libexecdir@/fwupdtool get-device-flags 2>/dev/null)"
+	flags="$(command fwupdtool get-device-flags 2>/dev/null)"
 	COMPREPLY+=( $(compgen -W "${flags}" -- "$cur") )
 }
 
 _show_firmware_types()
 {
 	local firmware_types
-	firmware_types="$(command @libexecdir@/fwupdtool get-firmware-types 2>/dev/null)"
+	firmware_types="$(command fwupdtool get-firmware-types 2>/dev/null)"
 	COMPREPLY+=( $(compgen -W "${firmware_types}" -- "$cur") )
 }
 
 
 _show_plugins()
 {
 	local plugins
-	plugins="$(command @libexecdir@/fwupdtool get-plugins 2>/dev/null)"
+	plugins="$(command fwupdtool get-plugins 2>/dev/null | sed -e '/^[a-zA-Z]/!d' -e '/ /d' -e 's/:$//')"
```

Comment:
Does https://github.com/fwupd/fwupd/pull/4177 help here? :)

Reply:
Yes. that helps. :)  One thing that fwupdtool prints something like `Loading... Idle...` to pipe `1` that should be removed until the first pattern `{` is seen, therefore `sed` still on the sofa. :(

Reply:
Maybe in json mode that should be turned off

Reply:
Agree: https://github.com/fwupd/fwupd/pull/4185